source/View.cpp

```diff
@@ -90,7 +96,10 @@ void View::xdg_surface_map([[maybe_unused]]struct wl_listener *listener, [[maybe
 	  }
 	  break;
 	case OpenType::floating:
-	  windowNode = wm::nullNode;
+	  {
+	    windowNode = wm::nullNode;
+	    server->openType = OpenType::dontCare;
```

Comment:
Do we need to set openType to dontCare even though it was on floating ?

Reply:
I thought this was better so that we may have a shortcut "open next window in floating" in the future.